codexctl/analysis.py

```diff
@@ -1,9 +1,13 @@
 import ext4
-import warnings 
+import warnings
 import errno
+import libconf
 
 from remarkable_update_image import UpdateImage
 from remarkable_update_image import UpdateImageSignatureException
+from remarkable_update_image.cpio import Archive
+from typing import Tuple, Optional, Dict

```

Comment:
These are all deprecated and should be replaced with `tuple`, `dict` and `| None` instead.